tests/responses_callbacks.py

```diff
@@ -36,7 +45,7 @@ def messages_callback(request):
     }
     headers = {'X-Request-Id': TEST_REQUEST_ID}
 
-    if getattr(request, 'headers', None).get('content-type', None) == 'application/x-www-form-urlencoded':
+    if getattr(request, 'headers', {}).get('content-type') == 'application/x-www-form-urlencoded':
```

Comment:
again, I'd rather see this stored as a variable before the conditional and that you explicitly state the logic behind these checks